### DISCUSSION TITLE
Welcome to the Community Discussions!

### DISCUSSION THREAD

**Discussion by fredygerman** [fredygerman/umeme-upo#5] [Announcements]
## Welcome to the Community Discussions!

👋 Hello and welcome to the community discussion page! We're excited to have you here and want to make sure you get the most out of your experience. Here's how you can participate effectively:

### Getting Started

- **Introduce Yourself**: Take a moment to introduce yourself to the community. Tell us who you are and what brings you here. We'd love to get to know you better.

- **Share Your Thoughts**: This is a space where you can share your ideas, ask questions, or start conversations on topics that interest you. Don't hesitate to start a new discussion thread or join existing ones.

- **Engage with Others**: Engaging with other community members is a key part of what makes this community thrive. Feel free to reply to discussions, offer your insights, or simply say hello.

- **Be Open-Minded**: We encourage an open and welcoming atmosphere. Remember that this is a community we build together. Respect and consider different perspectives.

Thank you for being a part of our community. We look forward to engaging with you, learning from your experiences, and building a stronger community together! 💪

**Comment by makindajack** [comment]
"Hello, 👋🏾 everyone! I'm Jackson Makinda "Chalii wa UI".

I'm here because of exciting project idea. @Era360 and I brainstormed this idea, and shortly after, @fredygerman initiated the repository. Consequently, we forked it and began our contributions. Stay tuned for more innovative ideas that we'll be sharing soon.

**Comment by fredygerman** [comment]
Thank you very much brother, Karibuni sana wote 🥳. Can't wait for your contributions also feel free to reach out to me anytime. Please show us some Ui magic on the UI issues #1 #9 #10.

**Reply by makindajack** [reply]
On it...

**Comment by Era360** [comment]
Hello, I'm Elia Mkumbo (@Era360 ) as @makindajack mentioned, we are all excited about this project.
I have a question though?
I recently bought home internet, how can it be of a contribution to the project by providing electricity status for people living around Tabata? @fredygerman

**Reply by fredygerman** [reply]
Hello @Era360 , Thank you very much for your interest and contribution to the project. 

Since you just got home internet you can be able to contribute to the project by adding your location. Currently i am out of esp-32 devices which are required to add new locations but  i have just made my order for more ESP-32 devices and they will be arriving in  1 to 2 weeks. 

Also there are 2 other people who have also reached out to add their locations so let's wait for the devices and get new locations added.

**Reply by fredygerman** [reply]
Hello @Era360, Good news 🥳. I just got 2 new Esp-32 devices, reach out to me. so I can send one to you and assist in setting up.

**Reply by Era360** [reply]
🥳That's good news. I have one concern though. My home internet came with a mini UPS. It can provide backup power for about 6 hours. I believe this will contribute to the inaccuracy of the platform since power can be out while the device continues to report that the power is on. How can we solve this challenge? @fredygerman

**Reply by fredygerman** [reply]
That's okay, since you don't have to plugin the device on your router. You can just plug it in on a line which is not connected to the UPS. @Era360

**Reply by Era360** [reply]
okay got it.
